drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -3999,6 +4095,10 @@ RID RenderingDeviceVulkan::framebuffer_create_multipass(const Vector<RID> &p_tex
 				size.width = texture->width;
 				size.height = texture->height;
 				size_set = true;
+			} else if (texture->usage_flags & TEXTURE_USAGE_VRS_ATTACHMENT_BIT) {
+				// If this is not the first attachement we assume this is used as the VRS attachment
```

Comment:
We could double check the size here and see if it is indeed 1/16th of the width and height, but I have a problem when we're writing to our VRS texture because in this case the size of the texture should be the size of the framebuffer.

I guess I could do the same thing as I do else where and only assume that we're using the texture as the VRS attachment if it is not the first attachment.